src/logic/dijkstra.ts

```diff
@@ -76,6 +76,7 @@ function getKeyOfMinValue(obj: Costs): number {
   return parseInt(Object.keys(obj).find((key) => obj[parseInt(key)] === minValue)!);
 }
 
-function arrayToObject(arr: Array<any>, defaultValue: any): any{
+function arrayToObject(arr: Array<any>, defaultValue: any): any {
+  // TODO unexpected use of comma operator - line below - col 61
```

Comment:
Yeah thats pain in the ass

Reply:
Idk, looks good to me. Doesn't work without comma. Gonna do eslint-ignore-next-line or w/e its called